test/integration/testConnectionWithOCSP.js

```diff
@@ -113,9 +109,14 @@ describe('Connection with OCSP test', function ()
         {
           connection.connect(function (err)
           {
-            assert.strictEqual(err.code, Errors.codes.ERR_SF_NETWORK_COULD_NOT_CONNECT);
-            assert.strictEqual(err.cause.code, Errors.codes.ERR_OCSP_INVALID_VALIDITY);
-            callback();
+            try {
+              assert.strictEqual(err.code, Errors.codes.ERR_SF_NETWORK_COULD_NOT_CONNECT);
+              assert.strictEqual(err.cause.code, Errors.codes.ERR_OCSP_INVALID_VALIDITY);
+              callback();
+            } catch (err) {
+              done(err);
```

Comment:
We don't need to check the error here?

Reply:
throwing the error is enough - any error should fail the test

Reply:
But we expect something specific right? This test would pass if for instance the error is connection refused, not ocsp error?

Reply:
done without any paramter or null or undefined means success, if parameter has a value it is an error cause